src/core/mac/mac_frame.hpp

```diff
@@ -981,16 +1075,55 @@ class Frame : public otRadioFrame
 
     static uint8_t GetKeySourceLength(uint8_t aKeyIdMode);
 
-    static bool IsDstAddrPresent(uint16_t aFcf) { return (aFcf & kFcfDstAddrMask) != kFcfDstAddrNone; }
-    static bool IsDstPanIdPresent(uint16_t aFcf);
-    static bool IsSequenceSuppressed(uint16_t aFcf)
+#if OPENTHREAD_CONFIG_MAC_MULTIPURPOSE_FRAME
+    static uint8_t GetFcfSize(uint16_t aFcf) { return IsShortFcf(aFcf) ? kShortFcfSize : kFcfSize; }
+#else
+    // clang-format off
+    static uint8_t GetFcfSize(uint16_t /* aFcf */) { return kFcfSize; }
+    // clang-format on
```

Comment:
Just curios why do we need this `clang-format off/on`?

Does `clang-format` mess with this line?

Reply:
Right, `script/make-pretty` CI check would fail on this line. Somehow, the script wanted to format this line as `static uint8_t                   GetFcf` (lots of spaces after the return type).

Reply:
I see. Thanks. `clang-format` can mess things up in some situations.